pkg/ccl/backupccl/backup_test.go

```diff
@@ -11204,3 +11204,44 @@ func TestRestoreMemoryMonitoringMinWorkerMemory(t *testing.T) {
 	actualFingerprints := sqlDB.QueryStr(t, "SHOW EXPERIMENTAL_FINGERPRINTS FROM TABLE restore.bank")
 	require.Equal(t, expectedFingerprints, actualFingerprints)
 }
+
+// TestBoundBackupExportOnRanges tests that export requests never get sent
+// across range boundaries, even if the ranges are empty.
+func TestBoundBackupExportOnRanges(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+	const numAccounts = 1
+
+	waitForSplits := make(chan struct{})
+	var responseCount atomic.Int32
+	params := base.TestClusterArgs{}
+	knobs := base.TestingKnobs{
+		DistSQL: &execinfra.TestingKnobs{
+			BackupRestoreTestingKnobs: &sql.BackupRestoreTestingKnobs{
+				RunBeforeBackupFlow: func(ctx context.Context) {
+					<-waitForSplits
+				},
+				RunAfterExportingSpanEntry: func(ctx context.Context, response *kvpb.ExportResponse) {
+					responseCount.Add(1)
+				},
+			},
+		},
+	}
+	params.ServerArgs.Knobs = knobs
+	tc, sqlDB, _, cleanupFn := backupRestoreTestSetupWithParams(t, singleNode, numAccounts, InitManualReplication, params)
```

Comment:
Not recommending this change. But when we set numAccounts = 1, I often wonder if it would be easier to 
use:

```
backuptestutils.StartBackupRestoreTestCluster(t, clusterSize, backuptestutils.WithParams(params))
```

to create an empty cluster and then create the table directly in the test. In this case your use of split at would require us to also add some tenant capabilities, so probably best to leave it as-is for now.